docs/installation.rst

```diff
@@ -72,10 +68,5 @@ Python 2.7 and 3.4+ are supported.
 
 * `matplotlib <http://matplotlib.org/>`__
 
-If you are using Astropy version 1.3 or later,
-then you have ``astropy.visualization.wcsaxes``.
-For older versions of Astropy, you have to install the separate package:
-`wcsaxes <https://github.com/astrofrog/wcsaxes>`__
-
-To work with the development version, you'll need Cython and a C compiler,
+To work with the development version, you'll need a C compiler,
```

Comment:
One doesn't actually need Cython installed anymore to build the package.  It gets pulled automatically to do the build now because it is PEP517-compliant, and is listed in pyproject.toml.  Best not to confuse people.  Just tell them how to install the package and leave out the underlying details of what happens under the hood.